src/adapter/src/coord.rs

```diff
@@ -2803,15 +2862,28 @@ impl Coordinator {
         let mut dataflows = Vec::new();
         let mut read_policies = BTreeMap::new();
         for entry in self.catalog.entries() {
-            let id = entry.id();
-            if let Some(plan) = self.catalog.try_get_physical_plan(&id) {
-                catalog_ids.push(id);
+            let gid = match entry.item() {
+                CatalogItem::Index(idx) => idx.global_id(),
+                CatalogItem::MaterializedView(mv) => mv.global_id(),
+                CatalogItem::ContinualTask(ct) => ct.global_id(),
+                CatalogItem::Table(_)
+                | CatalogItem::Source(_)
+                | CatalogItem::Log(_)
+                | CatalogItem::View(_)
+                | CatalogItem::Sink(_)
+                | CatalogItem::Type(_)
+                | CatalogItem::Func(_)
+                | CatalogItem::Secret(_)
+                | CatalogItem::Connection(_) => continue,
```

Comment:
Just confirming that we skip these because they're not compute dataflows?

Reply:
Exactly!